packages/examples/src/common/model.ts

```diff
@@ -0,0 +1,34 @@
+/* --------------------------------------------------------------------------------------------
```

Comment:
`model.ts` breaks the separation between node and client lands. Please move the client specific parts to `language-client-runner.ts` and the server related things to `language-server-runner.ts`.

The enum and interface declarations itself are fine.